style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
```

Comment:
this sounds a little awkward to me maybe change to.
`This page outlines the SCSS style guide for the Chassis project`